### PR TITLE
feat: Prevent unverified users to buy free tickets

### LINKED ISSUE #1881: Unverified email shouldn't be able to get free tickets
**Describe the bug**
Unverified email shouldn't be able to get ticket

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #1881

### CODE REVIEW

**Comment by liveHarshit** [comment]
@anhanh11001 Please review it.

File: app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt [hunk 1]
```diff
@@ -176,6 +195,13 @@ class TicketsFragment : Fragment() {
         ))
     }
 
+    private fun redirectTProfile() {
+        rootView.longSnackbar(getString(R.string.verify_your_profile))
+        findNavController(rootView).navigate(
+            TicketsFragmentDirections.actionTicketsToProfile()
```
[the remark this answers is not in the document]
Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt]: @anhanh11001

**Comment by mariobehling** [comment]
@CosmicCoder96 Shouldn't we prevent these kind of things on the server? Anyone who could access the APIs could circumvent this otherwise.

**Comment by ShridharGoel** [comment]
@liveHarshit This should be handled by the server instead.

**Comment by ShridharGoel** [comment]
@liveHarshit This should be handled by the server instead.

**Comment by kushthedude** [comment]
@liveHarshit @iamareebjamal  So we won't allow guest orders any more than?

**Comment by liveHarshit** [comment]
> @liveHarshit @iamareebjamal So we won't allow guest orders any more than?

Free orders*

**Comment by kushthedude** [comment]
> > @liveHarshit @iamareebjamal So we won't allow guest orders any more than?
> 
> Free orders*, yes

Recent server PR restricted paid orders , As far as free orders are considered , A recaptcha V.3 would work on FE

**Comment by iamareebjamal** [comment]
Yeah @ShridharGoel already mentioned in an issue earlier that it'll be handled on server and not here.

**Pull request closed without merging**